src/main/java/de/baleipzig/iris/ui/presenter/training/TrainingPresenter.java

```diff
@@ -24,6 +29,9 @@
     private INeuralNetTrainer<BufferedImage, Integer> trainer;
     private Map<BufferedImage, Integer> testData;
 
+    private ScheduledExecutorService progressService = new ScheduledThreadPoolExecutor(1);
+    Future<?> progressHandler;
```

Comment:
Future|Void| fänd ich besser ... die eckigen klammern gehen nicht in der Kommentarfunktion nicht